pillar.example

```diff
@@ -31,3 +33,24 @@ lighttpd:
       - status
       - userdir
       - usertrack
+
+  tofs:
+    # The files_switch key serves as a selector for alternative
+    # directories under the formula files directory. See TOFS pattern
+    # doc for more info.
+    # Note: Any value not evaluated by `config.get` will be used literally.
+    # This can be used to set custom paths, as many levels deep as required.
+    files_switch:
+      - id
+      - osfinger
+      - os
+      - os_family
+    # All aspects of path/file resolution are customisable using the options below.
+    # This is unnecessary in most cases; there are sensible defaults.
+    # path_prefix: template_alt
+    # dirs:
+    #   files: files_alt
+    #   default: default_alt
+    # files:
```

Comment:
This is the `files` that I believe should be changed.  Otherwise, there could be confusion between the two.

Reply:
there is a difference: the first files is just the name of the folder where the files lives.
a bether name could be folder? or dir,

I guess it the same with the other formulas, we use ```files``` in which we house files and templates, other configuration tools use separate places, one ```files``` one ```templates```